### PR TITLE
"How to Buy" - Chinese translation

### PR DESCRIPTION
**Pull request by 333x**
My taiga.getmonero.org name: ie333

### CODE REVIEW

**Comment by lh1008** [comment]
@333x could you please send me the email you used to open your taiga account to paintluis[at]protonmail[dot]com?

**Comment by 333x** [comment]
> @333x could you please send me the email you used to open your taiga account to paintluis[at]protonmail[dot]com?

I had sent the email a week ago.
My Taiga account email: ie333[at]icloud[dot]com 
My Taiga account username: ie333

**Comment by lh1008** [comment]
@blackey added as reviewer for this task.

https://taiga.getmonero.org/project/xmrhaelan-monero-public-relations/task/309

File: monero-outreach-docs/translations/zh-hans/archive/how_to_buy_zh-hans.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+| _**优点**_ | _**缺点**_ |
+|--|--|
+| 适用大额交易 | 不私密 |
+|  | 集中式的 |
+|  | 昂贵且不方便 |
+
+### 使用Paypal购买门罗币
+---
+
+Paypal、Apple Pay、Google Pay、Venmo或西联汇款这类型的服务并不私密，而且可能很昂贵。这些汇款服务也会有自己的服务条款和政策，可能会给你带来额外的风险。尤其是Paypal，XMR卖家需要小心，因为交易可能会被逆转，而PayPal的政策是在发生纠纷时站在买方一边。在一个骗局的例子中，爱丽丝同意以X美元从鲍勃那里购买1个XMR。爱丽丝通过Paypal向鲍勃发送了X美元，然后鲍勃向爱丽丝发送了1个XMR，爱丽丝向Paypal提出争议，并获得退还的X美元，最后她获得了不可逆转交易的门罗币。
```
Comment (jianjunchu) [inline monero-outreach-docs/translations/zh-hans/archive/how_to_buy_zh-hans.md:109]: “Paypal、Apple Pay、Google Pay、Venmo或西联汇款这类型的服务并不私密，而且可能很昂贵” -> "使用 Paypal、Apple Pay、Google Pay、Venmo或西联汇款这类型的服务并不会保护隐私，而且可能很昂贵"

**Review by jianjunchu — CHANGES_REQUESTED** [review]
Some sentences are not fluent, and suggestions for revision are given

**Comment by lh1008** [comment]
Pinging @333x :). The review has been done.

**Comment by 333x** [comment]
> Pinging @333x :). The review has been done.

Updated.

**Comment by lh1008** [comment]
@jianjunchu could you please approve the changes so we can merge them? :)

**Review by jianjunchu — APPROVED** [review]

**Review by lh1008 — APPROVED** [review]
Thank you @333x and  @jianjunchu 

This should be ready to merge.

**Pull request merged**